.github/workflows/code_size.yml

```diff
@@ -1,7 +1,6 @@
 name: Check code size
 
 on:
-  push:
```

Comment:
With this removed, will the code-size report still update when the PR is updated with addition commits, or force pushed?

Reply:
Yes: "*By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened*" ([ref](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request)), and `synchronize` is the event when someone updates the branch.

Pull request workflows don't run if there is a merge conflict, so we won't get a code size report any more until the conflict is resolved (although I think with the current setup the `push` report would likely be incorrect any time there's a merge conflict.)

Reply:
OK, great, thanks for checking.